src/modules/Dashboard/features/RespondentData/RespondentData.const.ts

```diff
@@ -12,6 +12,13 @@ export const defaultRespondentDataFormValues = {
   filterByIdentifier: false,
   identifier: [],
   versions: [],
+  summaryActivities: [],
```

Comment:
These entities were moved from `RespondentData form` to the `RespondentDataContextProvider` during the implementation of Activity Flow support in DataViz. So, maybe it's better to remove them from the form?